Mage.Sets/src/mage/cards/e/Epochrasite.java

```diff
@@ -45,7 +36,7 @@ public Epochrasite(UUID ownerId, CardSetInfo setInfo) {
                 new CastFromHandWatcher());
 
         // When Epochrasite dies, exile it with three time counters on it and it gains suspend.
-        this.addAbility(new DiesSourceTriggeredAbility(new EpochrasiteEffect()));
+        this.addAbility(new DiesSourceTriggeredAbility(new ExileSpellWithTimeCountersEffect(3,true)));
```

Comment:
style: space after comma between params. Also, you'll probably need to `setText()` here